resources/i18n/zh_TW/cura.po

```diff
@@ -2050,27 +2050,27 @@ msgstr "已排入隊列"
 #: /home/ruben/Projects/Cura/plugins/UM3NetworkPrinting/resources/qml/MonitorQueue.qml:67
 msgctxt "@label link to connect manager"
 msgid "Go to Cura Connect"
-msgstr ""
+msgstr "前往 Cura 連線"
```

Comment:
Connect shouldn't be translated.

Reply:
Fixed: 47d4ae2789d6962426662515dd0639ba5b2dcdb1